recipes/pcapplusplus/all/test_package/conanfile.py

```diff
@@ -1,18 +1,39 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.build import can_run
+from conans import CMake
 import os
+import shutil
 
 
 class PcapplusplusTestConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     generators = "cmake"
+    # Needed to make sure libpcap gets compiled
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+        if can_run(self) and self.settings.os == "Windows":
+            self.requires("libpcap/1.10.1")
```

Comment:
Why npcap is not enough and why libpcap is not listed as package requirement instead of in test_package?

Reply:
Because `wpcap.dll` that comes from the npcap driver is needed for the application to start.

The alternatives would be
* Configure the linker with `/DELAYLOAD:wpcap.dll` https://learn.microsoft.com/en-us/cpp/build/reference/delayload-delay-load-import?view=msvc-170 this will allow the application to start, but will crash if DLL is not found when needed.
   - Not sure if this will actually work, depends on test package logic, but worth a shot.
   - Works only with MSVC compiler
* Use replacement DLL as a mock: Libpcap compiled as shared.